### ISSUE TITLE
Question about offset

### ISSUE THREAD

**Issue by sangrockEG** [Pointcept/Pointcept#93]
First of all, thank you for releasing this wonderful project. 
It's really helpful for me.

By the way, I'm working on point cloud semantic segmentation on the S3DIS dataset.
Following your recommendation, I run "sh scripts/train.sh -g 4 -d s3dis -c semseg-pt-v2m2-0-base -n semseg-pt-v2m2-0-base" and get satisfactory performance (69~70).

However, I find something strange when tracking "offset."
I use batch size 12 with 4 GPUs, and the maximum number of points is 80000, as your default setting.
So, I thought the offset for each GPU should have three elements, like [A, A+B, A+B+C], and A,B,C<80000.

So, I simply put "print(coord.shape, offset)" at "pointcept/pointcept/models/point_transformer_v2/point_transformer_v2m2_base.py" just below line 559, "offset = data_dict["offset"].int().
 
But the result is as follows:

torch.Size([150930, 3]) tensor([100900, 150930], device='cuda:2', dtype=torch.int32)
torch.Size([123895, 3]) tensor([ 75930, 123895], device='cuda:0', dtype=torch.int32)
torch.Size([209377, 3]) tensor([136162, 209377], device='cuda:3', dtype=torch.int32)
torch.Size([221053, 3]) tensor([141053, 221053], device='cuda:1', dtype=torch.int32)

Here, I wonder why there are only two elements in the offset. I thought there should be three elements since the batch on each GPU has 12/4=3 point cloud samples.
Also, I wonder why the first element is sometimes larger than 80000. 
Since A+B or A+B+C could be larger than 80000, it is OK for the second or third element. But the first element should be bounded by 80000, isn't it?

I think the overall code runs perfectly, so I probably made some mistakes or wrong approaches for getting offset values.
Since I want to gather the seg_logit of each sample separately during training, obtaining the correct offset is crucial for me.
Your comment would be greatly appreciated.

**Comment by Gofinge** [comment]
Hi, check our code here: https://github.com/Pointcept/Pointcept/blob/main/pointcept/datasets/utils.py#L51

In our implementation of Mix3d, we just need to simply remove partial offset elements.

The strategy is controlled by mix_prob in our config, set to 0 can avoid mixing up two scenes.

**Comment by sangrockEG** [comment]
Thanks for your prompt reply!
Then, can you let me know how much the mixup strategy affects the performance?
Of course, I will try myself, but if the performance drop is too drastic, it would be sad :(

**Comment by Gofinge** [comment]
It doesn’t have a direct influence on mIoU (almost no influence), but I think it is good for model generation ability (more stable) which can be observed in validation loss. Consequently, I enable it in all my configs.

**Comment by sangrockEG** [comment]
Thanks again for your kind explanation.
I'll try.
